Translation/en/RedirectedResources/assets/abdata/h/list/01/personality_voice_c07/translation.txt

```diff
@@ -202,235 +202,235 @@
 そ、そこは……んっ、なんだ……体が疼いて……んっんっ……=T-That's... nn--, what... it's making my body throb... nnnnn......
-うん……もっと、おしりを弄ってくれ……=Yeah... play with my butt, even more......
-あぁん、もっとおしりを犯してくれ……=Aaahn, violate my butt even more...
+うん……もっと、おしりを弄ってくれ……=Yeah... play with my butt more......
+あぁん、もっとおしりを犯してくれ……=Aaahn, rape my butt even more...
 お、おしりが熱い……=M-My butt's so hot...
 あぁん……ち、力が入らない……=Aaahn... I-I'm going limp...
-これ……そう、もっとおしりを弄って……=This... that's it, play even more with my butt...
-ひゃああ……おしりが、変になる……=Hyaaah... my butt's, gonna go crazy...
-あっ……だめ、もう……おしりが、くっ……=Ah--... don't, I can't... my butt, kuh--...
-はぁはぁ……もう、イッちゃいそうだ……おしりで感じさせて=haaahaa... I'm gonna, cum... make me cum, from my butt.
-ば、馬鹿ッ！　これ以上は、駄目だ！=D-Dummy--! Stop it at that!
-おい、いい加減にしてくれないか？=Hey, can't you like, chill out with that?
+これ……そう、もっとおしりを弄って……=That... that's it, play even more with my butt...
+ひゃああ……おしりが、変になる……=Hyaaah... my butt's gonna go crazy...
+あっ……だめ、もう……おしりが、くっ……=Ah--... oh god, I can't... my butt, kuh--...
+はぁはぁ……もう、イッちゃいそうだ……おしりで感じさせて=*Pant pant*... I'm gonna, cum... make me cum from my butt.
+ば、馬鹿ッ！　これ以上は、駄目だ！=D-Dummy--! No more of that!
+おい、いい加減にしてくれないか？=Hey, can you maybe just chill out?
 私にだって、限度があるんだからな=Even I have my limits, you know?
 変態っ！　なんで、尻を触るんだ！？=Pervert--! Why are you touching my butt!?
-尻を触ろうとすんじゃねぇ=Don't touch my butt.
-次、尻を触ろうとしたら、ただじゃおかねぇからな=If you try touching my butt again, I won't let you go so easily.
+尻を触ろうとすんじゃねぇ=Don't try touching my butt.
+次、尻を触ろうとしたら、ただじゃおかねぇからな=Next time you try and touch my butt, you're gonna be in big trouble.
 あぁん！　ばっ、強すぎだ！=Aahn! Idi--, that's too much!
 んっ、だめ、やめろ！=Nn--, don't, stop!
 馬鹿ッ……やめろ……強いんだよ……=Idiot... stop it... that's too much...
-そこが、揉まれると……体が……=If you, rub me there... my body...
-あまり揉まれるのは、慣れてないんだけどよ……=I'm still not used to, being rubbed this much...
-いい……もっと、揉んでくれよ……=It's good... rub me more...
+そこが、揉まれると……体が……=If you rub me there... my body...
+あまり揉まれるのは、慣れてないんだけどよ……=I'm still not used to... being rubbed this much...
+いい……もっと、揉んでくれよ……=Good... rub me more...
 これ……いいかもな……=This... might be pretty nice...
-はぁん……可笑しな気分になるな……=Haan... I'm getting into a weird mood...
-んっ……お前に触られると、気持ちいな……=Nn--... It feels so good, when you touch me...
-そう、んっ……気分が良くなるな……=That's it, nn--... It feels really quite nice...
-あぁん……胸、感じてしまうな……=Aaahn... my chest, feels so good...
+はぁん……可笑しな気分になるな……=Haan... that makes me feel strange
+んっ……お前に触られると、気持ちいな……=Nn--... it feels good when you touch me, man...
+そう、んっ……気分が良くなるな……=That's it, nn--... that feels really quite nice...
+あぁん……胸、感じてしまうな……=Aaahn... my chest... feels so good...
 あぁ……胸、胸をもっと……=Aah... my chest, do my chest more...
 もっと尻を強く揉んでくれ……=Rub my butt even harder...
 はぁん……尻が……そ、そこをもっと……=Haan... my butt... t-that spot, do it more...
 うわっ！？　あ、あまり乳首をつつくな……=Uwah--!? D-Don't poke my nipples so much...
 乳首つつかれると、ビクンてなるな……=When you poke them, they twitch...
-お前につつかれると、気持ちいいな……=It feels really good, when you poke me...
+お前につつかれると、気持ちいいな……=It feels really good when you poke mtheme...
```

Comment:
Considering that it sounds like the player character is teasing the tomboy's nipples, I think "them" might work here.